### ISSUE TITLE
[textinput] Calling CurrentSuggestion() when there is no suggestion causes a panic

### ISSUE THREAD

**Issue by johnknott** [charmbracelet/bubbles#570]
**Describe the bug**

Calling `CurrentSuggestion()` when there is no suggestion causes a panic, and unless I am being daft, there is no public way to check if there is an available suggestion. I had to get round it with reflection to access a private field (`matchedSuggestions`).

**Setup**

 - OS: Arch Linux
 - Shell: fish
 - Terminal Emulator: foot
 - Terminal Multiplexer: none
 - Locale: en_GB.UTF-8

**To Reproduce**
1) go run `<source_file.go>`
2) press `<tab>` when there are no suggestions
3) panic

**Source Code**

```go
package main

import (
	"fmt"

	"github.com/charmbracelet/bubbles/textinput"
	tea "github.com/charmbracelet/bubbletea"
)

type model struct {
	textInput textinput.Model
}

func initialModel() model {
	ti := textinput.New()
	ti.Placeholder = "Start typing..."
	ti.ShowSuggestions = true
	ti.Focus()
	ti.SetSuggestions([]string{"apple", "banana", "cherry"})
	return model{textInput: ti}
}

func (m model) Init() tea.Cmd { return textinput.Blink }

func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
	var cmd tea.Cmd
	switch msg := msg.(type) {
	case tea.KeyMsg:
		if msg.Type == tea.KeyTab {
                        suggestion := m.textInput.CurrentSuggestion() // Panic here
			m.textInput.SetValue(suggestion)
		} else if msg.Type == tea.KeyCtrlC || msg.Type == tea.KeyEsc {
			return m, tea.Quit
		}
	}
	m.textInput, cmd = m.textInput.Update(msg)
	return m, cmd
}

func (m model) View() string {
	return fmt.Sprintf("Type a fruit:\n\n%s\n\n(esc to quit)\n", m.textInput.View())
}

func main() {
	if _, err := tea.NewProgram(initialModel()).Run(); err != nil {
		fmt.Printf("Error: %v", err)
	}
}
```

**Expected behavior**

`CurrentSuggestion()` to return ""

**Comment by ldez** [comment]
The problem has been fixed inside #473.

It's a part of [v0.19.0](https://github.com/charmbracelet/bubbles/releases/tag/v0.19.0)

**Comment by bashbunni** [comment]
Thank you for your report! This has since been fixed, so I will close this issue.
